File: javascript/Contagem.js

function contarNumeros() {

    let tamanho = parseInt(prompt("Digite o tamanho da lista: "))

    const numeros = [];
    for (let i = 0; i < tamanho; i++) {
        const valor = prompt(`Digite o valor ${i + 1}:`);
        numeros.push(Number(valor));
    }
    return `Você digitou ${numeros.length} números.`;
}

// // Chamada da Função

contarNumeros();

File: java/Contagem.java
package java;

import java.util.Scanner;

public class Contagem {
  public void contarNumeros() {
    Scanner scanner = new Scanner(System.in);
    System.out.print("Informe o tamanho do array: ");
    int tamanho = scanner.nextInt();
    int[] numeros = new int[tamanho];
    for (int i = 0; i < tamanho; i++) {
      System.out.print("Digite o valor " + (i + 1) + ": ");
      numeros[i] = scanner.nextInt();
    }
    System.out.println("Total de valores digitados: " + tamanho);
  }
}
